test/bdd/features/datalayer.feature

```diff
@@ -47,7 +47,20 @@ Feature: Data layer related features
     And DC imports "importers/xml_examples/Retail/01_Green_to_pink_shipment.xml" as GS1
     Given DC initiates the replication for last imported dataset
     And DC waits for last offer to get written to blockchain
-    Then DC manually calculated datasets data and root hashes matches ones from blockchain
+    Then DC manually calculated 2 datasets data and root hashes matches ones from blockchain
+
+  @second
+  Scenario: Dataset immutability II
+    Given I setup 1 node
+    And I start the node
+    And I use 1st node as DC
+    And DC imports "importers/xml_examples/Basic/01_Green_to_pink_shipment.xml" as GS1
+    Given DC initiates the replication for last imported dataset
+    And DC waits for last offer to get written to blockchain
+    And DC imports "importers/xml_examples/Basic/01_Green_to_pink_shipment_modified_event_timestamp.xml" as GS1
+    Given DC initiates the replication for last imported dataset
+    And DC waits for last offer to get written to blockchain
+    Then DC manually calculated 2 datasets data and root hashes matches ones from blockchain
```

Comment:
![](https://i.imgflip.com/eo62r.jpg)

Can we make 'Then DC manually calculated 2 datasets data and root hashes matches ones from blockchain' simpler?

Reply:
done